### DISCUSSION TITLE
Adding more events into the plugin

### DISCUSSION THREAD

**Discussion by WilliamRiker2701** [jamescl604/MSFSVoiceAttackPlugin#16] [Q&A]
Hi! First of all James thank you for this great work!

I've been testing with it with the objective of increasing the amount of compatible simconnect events and making it possible to do entire checklists. My goal is with the assistance of TTS phrases to create a virtual first officer. 
I'm banging against a brick wall at this moment tho, because whilst i defined new events like parking brakes and master battery toggle in events.cs with the right structure, when passing a command related to these events in the VoiceAttack log it reads that the context is unknown to the agent.
I tried everything, and i honestly don't get why the plugin does not recognize these events.

¿Anyone available to give a hand? 

I'm not a programming master at all, but if manage to pass these problems i plan to do a tedious work of adding a big chunk of new events and variables into the code.

**Comment by WilliamRiker2701** [comment]
Solved it!! i was having a silly issue with referencing Microsoft.FlightSimulator.SimConnect in VS.

Anyways, if i get the project working with lots of more events and variables i will report it here. Cheers!

**Reply by KevinMicknick** [reply]
Did you get this figured out?

**Reply by craigt47** [reply]
How did you fix this? I am having a similar issue in the build?

**Comment by jimmyap** [comment]
Does anyone have a working profile that can change nav 1 etc?
I can get the swap function to work but can't set the nav or radios!

**Reply by WilliamRiker2701** [reply]
Hey, I developed a tool that is able to change radios and do much much more.
Enter the discord and read instructions to download and install.
Any questions you can ask them in there
https://discord.com/invite/Tmzyba72WD
